companion/src/firmwares/customfunctiondata.h

```diff
@@ -71,6 +71,7 @@ enum AssignFunc {
   FuncBindExternalModule,
   FuncRacingMode,
   FuncDisableTouch,
+  FuncDisableAudioAmp,
```

Comment:
New functions must be added at the end of list to avoid breaking binary import or enum cross references

Reply:
done